3-platform/shopping-cart/db/postgres/01-create-shopping-cart.sql

```diff
@@ -0,0 +1,19 @@
+-- DROPPING TABLE IF EXISTS
+-- DROP TABLE IF EXISTS shopping_carts;
+
+-- CREATING TABLE "shopping_carts"
+CREATE TABLE IF NOT EXISTS shopping_carts
+(
+    id         VARCHAR(20) NOT NULL,
+    product_id VARCHAR(150)
+);
+
+-- SET INDEXES WHERE NEEDED
+CREATE INDEX idx_shopping_carts_id ON shopping_carts (id);
+
+-- SET COMMENT ON TABLE
+COMMENT ON TABLE shopping_carts IS 'Table containing shopping cart information';
+
+-- SET COMMENT ON COLUMN
+COMMENT ON COLUMN shopping_carts.id IS 'The cart unique identifier';
```

Comment:
Comment on shopping_carts.id is not actual anymore because we don't have PK.
'The cart identifier'

Reply:
I agree, it's done